cmd/nri-vsphere/nri-vsphere.go

```diff
@@ -46,7 +46,12 @@ func main() {
 	if err != nil {
 		config.Logrus.WithError(err).Fatal("failed to create client")
 	}
-	defer client.Logout(config.VMWareClient)
+	defer func() {
```

Comment:
I could have created a function, but I do not like that much when there is the defer calling a function hiding the error management. Wdyt?

Reply:
I think it's fine